csvcubed-pmd/tests/test-cases/LICENSE

```diff
@@ -0,0 +1,4821 @@
+
+
+
+
+
```

Comment:
I think this should be the raw text contents of the license instead of an HTML version. It's easier to work with in source control and clearer.